setup.py

```diff
@@ -3,8 +3,8 @@
 from distutils.core import setup
```

Comment:
Could you remove these changes to setup.py?

Reply:
Will do! Making requested changes now.